modules/ROOT/pages/managing-instances/billing.adoc

```diff
@@ -0,0 +1,39 @@
+[[aura-Billing]]
+= Billing
+:description: Consumption reporting allows Virtual Dedicated Cloud customers to closely monitor their consumption.
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Virtual Dedicated Cloud services are offered through prepaid consumption plans.
+Billing is based on usage, with credits deducted from the available balance each month.
+
+The consumption report, accessible in the Aura console’s Billing section, provides real-time insights into resource usage for the current project, including both running and paused states.
+It displays RAM usage in GB-hours and the equivalent cost in prepaid credits.
+You can also Export a CSV containing usage in a selected period.
+
+Available to Admins, the report helps track usage patterns over time to make informed resource allocation decisions.
+
+== Consumption report
+
+Monitor consumption in real-time in the Aura console’s Billing section.
+The report includes the following fields:
+
+* *Instance*
+* *Instance ID*
+* *Product name and pricing:* For more information, refer to https://console-preview.neo4j.io/pricing[Aura Pricing] for the specific product names and their prices. If a database is paused, its charge is reduced to 20% of the standard hourly rate.
+* *Billing status:* Can be `ongoing` (currently being billed) or `ended` (historical usage).
+* *Usage:* Based on the time databases run and the memory consumed.
+* *Units of measure:* For many products, GB-hours usage is calculated by multiplying the number of hours a database is running (whether actively used or not) by the memory size in gigabytes (GB). 
+The total usage for the selected period is displayed in GB-hours along with the equivalent cost in credits.
+Note that some products are not measured in GB-hours.
+* *Credits consumed:* Used credits.
```

Comment:
Self-explanatory.
I wonder if a bullet list is useful here?
The way the original docs was structured makes more sense to me, where you only highlight things that are not obvious, such as product name and pricing, billing status, and units of measure.